packages/elements/src/components/ino-accordion/ino-accordion.scss

```diff
@@ -0,0 +1,38 @@
+@use 'base/typography-new' as typography;
+@use 'base/new-theme' as theme;
+
+.ino-accordion {
+  &:hover {
+    background: #f6fafe;
```

Comment:
We should use the variables of the colors instead if its hex code. If it doesn't exist in the `new-theme` we should add it (in this case `p-01`) and use it in our component

![image](https://user-images.githubusercontent.com/22963121/217881851-4814d01e-ad1f-4a5b-9aff-0c4b7b68f3cd.png)